### DISCUSSION TITLE
v1.15.2

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#834] [Announcements]
- Session cookies are no longer secure by default, as it breaks any LinkAce running under http://
- Users can now search for links without tags or lists (#824)
- Dependencies were updated, including security patches

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.15.2'>v1.15.2</a>.</em>

**Comment by secnum** [comment]
Sorry, I hadn't thought of those who don't encrypt their http traffic. Perhaps a security section should be added to the documentation to indicate which security features can be configured.

**Reply by Kovah** [reply]
No worries, I hadn't thought of it too. I'll see to include this in the updated docs for v2.
